packages/plugins/dashboard-time-track-angular-ui/README.md

```
@@ -0,0 +1,56 @@
+- **Lazy Loading**: Integrated with `defineDeclarativePlugin` for on-demand loading.
+- **Dynamic Routing**: Registers the `time-tracking` route under the `page-sections` location.
+- **Dashboard Integration**: Automatically adds a "Time Tracking" tab to the Gauzy dashboard tabset.
+- **I18n Support**: Full localization support for 13+ languages.
+- **Permissions**: Respects `ADMIN_DASHBOARD_VIEW` and `TIME_TRACKING_DASHBOARD` permissions.
+
+## Scripts
+
+### Building
+
+To build the library, run:
+```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)